build/cloudbuild-master.yaml

```diff
@@ -24,9 +24,13 @@ steps:
   args: ['build', '--target', 'pre-build', '-t', 'gcr.io/$PROJECT_ID/forseti-prebuild:master', '.']
 - name: 'gcr.io/cloud-builders/docker'
   id: Build forseti-security master container image
-  args: ['build', '-t', 'gcr.io/$PROJECT_ID/forseti:master', '.']
+  args: ['build',  '--target', 'runtime', '-t', 'gcr.io/$PROJECT_ID/forseti:master', '.']
+- name: 'gcr.io/cloud-builders/docker'
+  id: Build forseti-server master container image
+  args: ['build', '--target', 'forseti-server', '-t', 'gcr.io/$PROJECT_ID/forseti-server:master', '.']
 
 images:
+- gcr.io/$PROJECT_ID/forseti-server:master
 - gcr.io/$PROJECT_ID/forseti:master
 - gcr.io/$PROJECT_ID/forseti-base:master
 - gcr.io/$PROJECT_ID/forseti-prebuild:master
```

Comment:
Nothing specific to this PR, but I am curious if we really need to publish the base and prebuild images?

Reply:
Honestly, not sure.  I thought it might speed up builds in Cloud Build, but it isn't.  It's worth considering removing them.